code/modules/client/preferences.dm

```diff
@@ -547,6 +547,9 @@ GLOBAL_LIST_EMPTY(preferences_datums)
 				else
 					HTML += "<font color=red>[rank]</font></td><td><font color=red><b> \[NON-HUMAN\]</b></font></td></tr>"
 				continue
+			if((rank == "AI") && !(config.allow_ai))
+				HTML += "<font color=red>[rank]</font></td><td><font color=red><b> \[DISABLED\]</b></font></td></tr>"
+				continue

```

Comment:
@bgobandit then maybe you should specify that.

We are all under the assumption this is a bug where somebody can still roll ai if it's disabled after initializations.

Thats how you and the bug report seemed to make it sound.

If selection code already properly handles that, and this is just about making it unselectable, that should be specified in the pr body.